packages/react/src/components/Breadcrumb/BreadcrumbItem.tsx

```
@@ -0,0 +1,130 @@
+    ref: React.Ref<HTMLLIElement>
+  ) {
+    const prefix = usePrefix();
+    const className = cx({
+      [`${prefix}--breadcrumb-item`]: true,
+      // We set the current class only if `isCurrentPage` is passed in and we do
+      // not have an `aria-current="page"` set for the breadcrumb item
+      [`${prefix}--breadcrumb-item--current`]:
+        isCurrentPage && ariaCurrent !== 'page',
+      [customClassName]: !!customClassName,
+    });
+
+    const child = children as React.FunctionComponentElement<any>;
+    if (
+      child.type &&
+      child.type.displayName !== undefined &&
+      child.type.displayName.includes('OverflowMenu')
+    ) {
+      const horizontalOverflowIcon = (
+        <OverflowMenuHorizontal className={`${prefix}--overflow-menu__icon`} />
+      );
+      return (
+        <li className={className} {...rest}>
+          {React.cloneElement(child, {
+            menuOptionsClass: `${prefix}--breadcrumb-menu-options`,
+            menuOffset: { top: 10, left: 59 },
+            renderIcon: () => horizontalOverflowIcon,
+          })}
+        </li>
+      );
+    }
+
+    if (typeof children === 'string') {
+      return (
+        <li className={className} ref={ref} {...rest}>
+          {href ? (
+            <Link href={href} aria-current={ariaCurrent}>
+              {children}
+            </Link>
+          ) : (
+            <Text className={`${prefix}--link`}>{children}</Text>
+          )}
+        </li>
+      );
+    }
+
+    return (
+      <li className={className} ref={ref} {...rest}>
+        {React.cloneElement(child, {
+          'aria-current': ariaCurrent,
+          className: cx(`${prefix}--link`, child.props.className),
+        })}
+      </li>
+    );
+  });
+
+BreadcrumbItem.displayName = 'BreadcrumbItem';
+
+BreadcrumbItem.propTypes = {
+  // @ts-expect-error - v12 TODO: BREAKING: This should match AriaAttributes['aria-current']
+  'aria-current': PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
```

This definition is to broad for 'aria-current'.  Per my discussion [here](https://ibm-cloudplatform.slack.com/archives/C03C8VASVED/p1697813662395289) I have decided to simply suppress the error in order to not introduce breaking changes in the typing additions.  I have opened #15034 where I will address this issue.

---

I have created #15052 to address the Incorrect PropTypes to be accepted at a later time as it is a Breaking change.